Source/HAConnection.swift

```diff
@@ -180,4 +180,9 @@ public protocol HAConnection: AnyObject {
         initiated: @escaping SubscriptionInitiatedHandler,
         handler: @escaping (HACancellable, T) -> Void
     ) -> HACancellable
+
+    /// Send audio to Assist
+    /// - Parameters:
+    ///   - request: The data request containing sttBinaryHandlerId and data (as base64 string) to be written
+    func sendSttAudio(_ request: HARequest)
```

Comment:
The HARequest should have enough that the normal 'send' works. You can make it invoke the completion handlers on send rather than success and the rest of the paradigm should work.